### PR TITLE
feature/STJS-672-jwt-generator

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: packages/jwtgenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# `jwtgenerator`
+
+> Generator of JSON Web Token
+
+## Usage
+```
+import { jwtgenerator } from '@securetrading/jwtgenerator';
```
Comment (wirus15) [inline packages/jwtgenerator/README.md:7]: this should be available from the CLI:
```
$: ./jwtgenerator --payload payload.json --secret foobar12345 --iss testiss
```
or
```
$: ./jwtgenerator --payload payload.json --secretFile secret.txt --iss testiss
```
Than we could use it in npm scripts, like:
```
$: npm run jwtgen

Comment (pgs-gmartowski) [reply packages/jwtgenerator/README.md:7]: ok, done

File: packages/jwtgenerator/cli.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+import { jwtgenerator } from './lib/jwtgenerator';
+import * as config from './config.json';
+
+const { payload, iss, secret } = config;
+// @ts-ignore
+const jwt = jwtgenerator(payload, secret, iss);
+console.log(jwt);
```
Comment (pgs-gmartowski) [inline packages/jwtgenerator/cli.ts:10]: info: left this for CLI usage

File: packages/jwtgenerator/config.json.dist [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "iat": 1592298893548,
+  "iss": "",
+  "secret": "dupa dua",
```
Comment (wirus15) [inline packages/jwtgenerator/config.json.dist:4]: pls don't 😅

Comment (pgs-gmartowski) [reply packages/jwtgenerator/config.json.dist:4]: why not 😃 ? ok I change it ;)

Comment (pgs-gmartowski) [reply packages/jwtgenerator/config.json.dist]: done 😞

**Comment by wirus15** [comment]
@pgs-gmartowski according to what we discussed on the call, I'd like to see 2 more features here:
1. Move the `config.json` and `config.json.dist` from this repo to the `js-payements`. They could also be renamed to be more meaningful, e.g. `jwtdata.json` and `jwtdata.json.dist`
2. Load the jwt data dynamically based on the cli args, something like:
```
$: ./jwtgenerator jwtdata.json
```
3. Add the `bin` section to the `package.json` so that the script will be symlinked after the `npm install` into `./node_modules/.bin/...`. With this it could be run from the js-payments project like `npx jwtgen jwtdata.json`. It would be easier then to put it into `js-paymens` package.json into scripts section.

**Pull request merged**